components/application-authenticators/org.wso2.carbon.identity.application.authenticator.social/src/main/java/org/wso2/carbon/identity/application/authenticator/social/google/GoogleOAuth2Authenticator.java

```diff
@@ -195,43 +189,14 @@ protected String getAuthenticateUser(OAuthClientResponse token) {
     }
 
     /**
-     * Get Subject Attributes
-     *
-     * @param token
-     * @return
+     * Get google user claim uri.
+     * @param token OAuth client response.
+     * @return User claim uri.
      */
     @Override
-    protected Map<ClaimMapping, String> getSubjectAttributes(
-            OAuthClientResponse token) {
-
-        Map<ClaimMapping, String> claims = new HashMap<ClaimMapping, String>();
-
-        try {
-
-                String json = sendRequest(token.getParam(GoogleOAuth2AuthenticationConstant.GOOGLE_USERINFO_ENDPOINT),
-                        token.getParam(OIDCAuthenticatorConstants.ACCESS_TOKEN));
-            if (StringUtils.isNotBlank(json)) {
-                Map<String, Object> jsonObject = JSONUtils.parseJSON(json);
-
-                if (jsonObject != null) {
-                    for (Map.Entry<String, Object> entry : jsonObject.entrySet()) {
-                        claims.put(ClaimMapping.build(entry.getKey(),
-                                entry.getKey(), null, false), entry.getValue()
-                                .toString());
-                        if (log.isDebugEnabled()) {
-                            log.debug("Adding claim from end-point data mapping : " + entry.getKey() + " - " +
-                                    entry.getValue());
-                        }
-
-                    }
-                }
+    protected String getUserClaimUri(OAuthClientResponse token) {
 
-            }
-        }catch (Exception e) {
-            log.error("Error occurred while accessing google user info endpoint", e);
-        }
-
-        return claims;
+        return token.getParam(GoogleOAuth2AuthenticationConstant.GOOGLE_USERINFO_ENDPOINT);
```

Comment:
Return value is correct, but method name is wrong. See above comment for explanation.